cmd/kubeapps-apis/proto/kubeappsapis/plugins/kapp_controller/packages/v1alpha1/kapp_controller.proto

```diff
@@ -4,15 +4,30 @@ option go_package = "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/
 
 import "google/api/annotations.proto";
 import "kubeappsapis/core/packages/v1alpha1/packages.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
 
 service PackagesService {
```

Comment:
Question here, @absoludity : is it possible to safely change this service name to something like `kappController_PackagesService` ? 

The generator is not able to deal with duplicated service names and we have to manually disambiguate the operation_id

Reply:
Yes, I think KappControllerPackagesService should be fine. The only restriction `buf lint` puts on is that it needs to end in `Service`, from memory. Feel free to update to that (and Fluxv2PackagesService) and see if that fixes it before landing. Check buf lint too.